app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -397,4 +397,6 @@ enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
     SET_AS_DEFAULT_PROMPT_CLICK("m_set-as-default_prompt_click"),
     SET_AS_DEFAULT_PROMPT_DISMISSED("m_set-as-default_prompt_dismissed"),
     SET_AS_DEFAULT_IN_MENU_CLICK("m_set-as-default_in-menu_click"),
+
+    MALICIOUS_SITE_DETECTED_IN_IFRAME("m_malicious-site-protection_iframe-loaded"),
```

Comment:
why is this one here if we have `MaliciousSitePixelName`? this should be inside malicious module.

Reply:
Because `MaliciousSitePixelName` belongs to the MSP implementation module, and the MSP module knows nothing about iframes, that happens on the browser level